my_game.py

```diff
@@ -297,31 +283,64 @@ def destroy(self):
         self.kill()
 
 
-class GameState(Enum):
+class IntroView(arcade.View):
     """
-    the state of the game: INTRO, IN_GAME or GAME_OVER
+    View for the intro screen.
     """
 
-    INTRO = auto()
-    IN_GAME = auto()
-    GAME_OVER = auto()
+    def __init__(self):
 
+        super().__init__()
 
-class MyGame(arcade.Window):
+        self.play_button = None
+        self.title_graphics = None
+
+        arcade.set_background_color(arcade.color.AMAZON)
+
+    def on_show_view(self):
+
+        self.title_graphics = arcade.load_texture("images/UI/asteroidsTitle.png")
+        self.play_button = arcade.load_texture("images/UI/asteroidsStartButton.png")
```

Comment:
Maybe we could set those variables in `__init__`, and then not have `on_show_view()`?